build.sbt

```diff
@@ -46,9 +46,11 @@ lazy val framework =
       case _ =>
         Seq()
     }) //workaround for scala/scala-dev#249
-    .settings(aggregatedSetting(sources in(Compile, doc)),
-              aggregatedSetting(dependencyClasspath in(Compile, doc)),
-              publishArtifact := false)
+    /*.settings(
+      (sources / aggregate) in (Compile, doc),
+      aggregatedSetting(dependencyClasspath in(Compile, doc)),
+      publishArtifact := false
+    )*/
```

Comment:
Is this something we can nuke now?

Reply:
Ah, yes, I had planned to replace this with the unidoc plugin but under either implementation I'm getting..... fun errrors:

```
[error] (lift-mapper / Compile / doc) java.net.URISyntaxException: Illegal character in path at index 45: http://www.scala-lang.org/api/SettingKey(This / This / This / version)/scala/xml/NodeSeq.html
```